script.embuary.helper/addon.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.embuary.helper" name="Embuary Helper" version="1.0.1" provider-name="sualfred">
+	<requires>
+		<import addon="xbmc.python" version="2.1.0"/>
```

Comment:
This will break the Krypton backwards compatiblity. That's why I used the ABI version. There is no fancy stuff included that requires a version bump. 

I cannot change it, because I also offer the skin + the helper in my private and Emby repo and it's a dependency of the Krypton version.

Reply:
Then change it at least to 2.25.0.

Reply:
Ok. Done